pkg/occlient/occlient_test.go

```diff
@@ -15,6 +15,110 @@ import (
 	ktesting "k8s.io/client-go/testing"
 )
 
+func TestGetPVCNameFromVolumeMountName(t *testing.T) {
+	type args struct {
+		volumeMountName string
+		dc              *appsv1.DeploymentConfig
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Test case : 1",
+			args: args{
+				volumeMountName: "test-pvc",
+				dc: &appsv1.DeploymentConfig{
+					Spec: appsv1.DeploymentConfigSpec{
+						Selector: map[string]string{
+							"deploymentconfig": "test",
+						},
+						Template: &corev1.PodTemplateSpec{
+							Spec: corev1.PodSpec{
+								Containers: []corev1.Container{
+									{
+										Name: "test",
+										VolumeMounts: []corev1.VolumeMount{
+											{
+												MountPath: "/tmp",
+												Name:      "test-pvc",
+											},
+										},
+									},
+								},
+								Volumes: []corev1.Volume{
+									{
+										Name: "test-pvc",
+										VolumeSource: corev1.VolumeSource{
+											PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+												ClaimName: "test-pvc",
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			want: "test-pvc",
+		},
+		{
+			name: "Test case : 2",
```

Comment:
Would be good to have a descriptive test name, eg. `Test case 2: Non-existent PVC`

Reply:
added names for test cases, ready for review now